r/DESCRIPTION

```diff
@@ -3,6 +3,7 @@ Title: R Integration to 'Apache' 'Arrow'
 Version: 0.0.0.9000
 Authors@R: c(
     person("Romain", "François", email = "romain@rstudio.com", role = c("aut", "cre")),
+    person("Javier", "Luraschi", email = "javier@rstudio.com", role = c("ctb")),
```

Comment:
@romainfrancois Lol, I don't think I deserve this yet... so feel free to remove be before going to CRAN... unless I ended up contributing something more significant, in any case, thanks!